web/src/lib/components/login/Modal.svelte

```diff
@@ -41,14 +39,10 @@
         if (typeof onLoginFunction === 'function') {
             onLogin = onLoginFunction;
         }
-
-        login.startCheckingLogin();
     }
 
     export function hide() {
         open = false;
-
-        login.stopCheckingLogin();
```

Comment:
I see that you moved this to the LN component's `onDestroy`, but testing this, if you close the login modal, it will be asking the server forever.